frontend/model/settings/vuexModule.js

```diff
@@ -70,6 +70,12 @@ const mutations = {
     state.increasedContrast = isChecked
   },
   setNotificationEnabled (state, enabled) {
+    if (state.notificationEnabled !== enabled) {
```

Comment:
Should this be `if (state.notificationEnabled !== enabled && enabled)` ?

Reply:
No, because this will also report the subscription, which can be used to remove subscriptions from the server.